src/locale/messages.en-US.xlf

```diff
@@ -402,6 +402,46 @@
         <source>notification-icon.toggle-panel.tooltip</source>
         <target state="new">Click to toggle notification panel</target>
       </trans-unit>
+      <trans-unit id="3299373246271290710" datatype="html">
+        <source>action-triggers.action.label</source>
+        <target state="new">Action</target>
+      </trans-unit>
+      <trans-unit id="719177318571875364" datatype="html">
+        <source>common.form-required-field.button-tooltip</source>
+        <target state="new">You have not filled in a required field</target>
```

Comment:
Our messages should not "blame" the user. An alternate phrasing that guides the user could be: `Please fill out the required field(s).`

If we want to just point out that is not been correctly filled out, then it could be something like: `One or more required fields have not been filled out.`